content/guides/references/migration-guide.md

```diff
@@ -310,6 +310,69 @@ cy.get('button')
    .then(res => { ...handle response... })
 ```
 
+#### `.within()`
+
+[`.within()`](/api/commands/within) now throws an error if it is passed multiple
```

Comment:
```suggestion
The [`.within()`](/api/commands/within) command now throws an error if it is passed multiple
```

Reply:
Applied this change to all the bits in this section (.invoke() and .should() were phrased the same way).